Code/Tools/AssetProcessor/native/utilities/BuilderManager.inl

```diff
@@ -82,10 +82,12 @@ namespace AssetProcessor
         }
 
         auto params = BuildParams(task.c_str(), modulePath.c_str(), "", jobRequestFile, jobResponseFile);
+        AZStd::string paramString;
+        AzFramework::StringFunc::Join(paramString, params.begin(), params.end(), " ");
```

Comment:
The `<AzCore/StringFunc/StringFunc.h>` include should be included at the top of the .inl file.

Also can you change this to use `AZ::StringFunc::Join`